From 6f235a2b59542565767fcc5895c270794f0c5910 Mon Sep 17 00:00:00 2001
From: Chang Yu-heng <mr.changyuheng@gmail.com>
Date: Mon, 21 Sep 2015 22:37:59 +0800
Subject: [PATCH 1/1] Disable epoll

---
 pyconfig.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyconfig.h b/pyconfig.h
index 0b5e2ca..431d5cc 100644
--- a/pyconfig.h
+++ b/pyconfig.h
@@ -188,7 +188,7 @@
 #define HAVE_DYNAMIC_LOADING 1
 
 /* Define if you have the 'epoll' functions. */
-#define HAVE_EPOLL 1
+/* #undef HAVE_EPOLL */
 
 /* Define to 1 if you have the `erf' function. */
 #define HAVE_ERF 1
@@ -732,7 +732,7 @@
 /* #undef HAVE_SYS_DIR_H */
 
 /* Define to 1 if you have the <sys/epoll.h> header file. */
-#define HAVE_SYS_EPOLL_H 1
+/* #undef HAVE_SYS_EPOLL_H */
 
 /* Define to 1 if you have the <sys/event.h> header file. */
 /* #undef HAVE_SYS_EVENT_H */
-- 
1.9.1

